scripts/dbsetup.sql

```diff
@@ -54,6 +56,12 @@ CREATE TABLE healthit_products (
     CONSTRAINT healthit_product_info UNIQUE(name, version)
 );
 
+CREATE TABLE endpoint_organization (
+    endpoint_id INT REFERENCES fhir_endpoints (id),
+    organization_id INT REFERENCES npi_organizations (id),
```

Comment:
how easy/hard would it be to add the match score here? another PR for the future or an easy addition?